app/src/main/java/com/quran/labs/androidquran/ui/fragment/AyahPlaybackFragment.java

```diff
@@ -58,13 +64,45 @@ public View onCreateView(LayoutInflater inflater,
     startAyahSpinner = view.findViewById(R.id.start_ayah_spinner);
     endingSuraSpinner = view.findViewById(R.id.end_sura_spinner);
     endingAyahSpinner = view.findViewById(R.id.end_ayah_spinner);
-    repeatVerseSpinner = view.findViewById(R.id.repeat_verse_spinner);
-    repeatRangeSpinner = view.findViewById(R.id.repeat_range_spinner);
     restrictToRange = view.findViewById(R.id.restrict_to_range);
     applyButton = view.findViewById(R.id.apply);
     applyButton.setOnClickListener(mOnClickListener);
+    repeatVersePicker = (NumberPicker) view.findViewById(R.id.repeat_verse_picker);
+    repeatRangePicker = (NumberPicker) view.findViewById(R.id.repeat_range_picker);
+    int defaultVerseRepeat = 3;
+    int defaultRangeRepeat = 7;
 
     final Context context = getActivity();
+
+    // Using Eastern Arabic Numerals
+    isArabicNames = QuranSettings.getInstance(context).isArabicNames();
+    if (isArabicNames){
+      String[] arNums = {"٩٩","٩٨","٩٧","٩٦","٩٥","٩٤","٩٣","٩٢","٩١","٩٠","٨٩","٨٨","٨٧","٨٦","٨٥","٨٤","٨٣","٨٢","٨١","٨٠","٧٩","٧٨","٧٧","٧٦","٧٥","٧٤","٧٣","٧٢","٧١","٧٠","٦٩","٦٨","٦٧","٦٦","٦٥","٦٤","٦٣","٦٢","٦١","٦٠","٥٩","٥٨","٥٧","٥٦","٥٥","٥٤","٥٣","٥٢","٥١","٥٠","٤٩","٤٨","٤٧","٤٦","٤٥","٤٤","٤٣","٤٢","٤١","٤٠","٣٩","٣٨","٣٧","٣٦","٣٥","٣٤","٣٣","٣٢","٣١","٣٠","٢٩","٢٨","٢٧","٢٦","٢٥","٢٤","٢٣","٢٢","٢١","٢٠","١٩","١٨","١٧","١٦","١٥","١٤","١٣","١٢","١١","١٠","٩","٨","٧","٦","٥","٤","٣","٢","١"};
```

Comment:
instead of this, can we just do:

```
val numberFormatter = NumberFormat.getInstance(Locale.ARABIC);
numberFormatter.format(number)
```

instead of having this array of numbers? as a matter of fact, guessing if we don't pass the locale, it gets it from the default locale, which we should be overriding to Arabic already.